QueryCiv3/BiqSections/Unit.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct UNIT
+    {
+        public int Length;
+
+        private fixed byte Text[32];
+        public string Name { get => Util.GetString(ref this, 4, 32); }
+
+        public int OwnerType; // 0: None, 1: Barbarian, 2: Civ, 3: Player
+        public int ExperienceLevel;
+        public int Owner;
+        public int Unit;
+        public int AIStrategy;
+        public int X;
+        public int Y;
+
+        private fixed byte Text2[57];
+        public string PTWName { get => Util.GetString(ref this, 64, 57); }
```

Comment:
Another place where Unit is confusing.  Checking my editor code, PTWName is what is used (in PTW+) for custom-named units.  I _believe_ the "regular" name is unused (can you set custom unit names in Vanilla?  I am not sure you can).  It might be worth naming this variable "CustomUnitName", or similar, with a comment that it's the "PTWUnitName".  Or perhaps just adding a comment that this one, not Name, is the one that is used.  Or maybe a comment on Name saying that this one is used instead.

Basically, something somewhere to indicate that this one is the one that is used.